pkg/cmd/server/admin/default_certs.go

```diff
@@ -82,6 +82,18 @@ func DefaultAPIClientCerts(certDir string) []ClientCertInfo {
 		DefaultClusterAdminClientCertInfo(certDir),
 		DefaultRouterClientCertInfo(certDir),
 		DefaultRegistryClientCertInfo(certDir),
+		DefaultProxyClientCertInfo(certDir),
+	}
+}
+
+func DefaultProxyClientCertInfo(certDir string) ClientCertInfo {
```

Comment:
The point of this cert's subject is to be verified by pods. I wouldn't expect random applications to work well with the Subject we generate to embed a group.